docs/Axis-Title-menu-layout.md

```diff
@@ -0,0 +1,81 @@
+# Axis Title menu layout
+Click on the axis title to open the Axis Title menu. This offers several functionalities, as shown below, enabling users to visualize data in a way that best aligns with their objectives.
+
+![Axis Title Menu Layout- final](https://github.com/user-attachments/assets/f4853b09-806a-451b-845b-d89acdfbc495)
```

Comment:
Update the image to show "Requested Features vs total features" to make things consistent

Reply:
Updated gif